defaults/main.yml

```
@@ -71,3 +71,6 @@ haproxy_backends: []
 haproxy_frontends: []
 haproxy_listen: []
 haproxy_userlists: []
+
+haproxy__log_rate_limit_interval_sec: 30
+haproxy__log_rate_limit_burst: 10000
```

Please only use a single `_` as this is the current style used in this role:
```suggestion
haproxy_log_rate_limit_interval_sec: 30
haproxy_log_rate_limit_burst: 10000
```